### PR TITLE
fix: fix docstring code block formatting

### PR DESCRIPTION
**Pull request by busunkim96**
Addresses issues identified in 219067629.

### CODE REVIEW

File: gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/rest.py.j2 [hunk 1]
```diff
@@ -66,7 +66,7 @@ class {{ service.name }}RestInterceptor:
     These use cases and more can be enabled by injecting an
     instance of a custom subclass when constructing the {{ service.name }}RestTransport.
 
-    .. code-block:
+    .. code-block: python
```
Comment (dandhlee) [inline gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/rest.py.j2:69]:
```suggestion
    .. code-block:: python
```

File: gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2 [hunk 2]
```diff
@@ -66,7 +66,7 @@ class {{ service.name }}RestInterceptor:
     These use cases and more can be enabled by injecting an
     instance of a custom subclass when constructing the {{ service.name }}RestTransport.
 
-    .. code-block:
+    .. code-block: python
```
Comment (dandhlee) [inline gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2:69]:
```suggestion
    .. code-block:: python
```

File: gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/rest.py.j2 (hunk 1 above)
Comment (dandhlee) [reply gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/rest.py.j2:69]: I don't see this being reflected in the Goldens below, so I'm not sure whether this should come with 2 colon characters or just one...

File: gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2 (hunk 2 above)
Comment (dandhlee) [reply gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2:69]: ditto

**Review by dandhlee — COMMENTED** [review]
Minor nit, not sure if this is a real concern. Please see comment below.

**Comment by busunkim96** [comment]
@dandhlee Thank you for catching that! The REST files aren't included in the goldens (I think there used to be some blocker to adding them, not sure if that still applies).

**Review by dandhlee — APPROVED** [review]
LGTM! Thank you :D

**Pull request merged**